CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Malli is in well matured [alpha](README.md#alpha).
 * **BREAKING**: `:gen/fmap` property requires its schema to create a generator.
   * previous behavior defaulted to a `nil`-returning generator, even if the schema doesn't accept `nil`
   * use `:gen/return nil` property to restore this behavior
+  * Decode map keys into keywords for `[:map` schemas in `json-transformer` [#1135](https://github.com/metosin/malli/issues/1135)
```

Comment:
This line is now nested under the `:gen/fmap` change, even though it's not related. Please add it as a top-level bullet point at the top of the list.